cpp/src/slice2py/PythonUtil.cpp

```diff
@@ -1211,7 +1211,7 @@ Slice::Python::CodeVisitor::writeOperations(const InterfaceDefPtr& p, Output& ou
         out << epar << operationReturnTypeHint(operation, MethodKind::Dispatch) << ":";
         out.inc();
 
-        writeDocstring(operation, MethodKind::Dispatch, out);
+        writeOpDocstring(operation, MethodKind::Dispatch, out);
```

Comment:
We had 3 overloads all named `writeDocString`, I gave them all different names.
Doing this actually revealed a useless `optional` parameter!